app/components/views/GovernancePage/Proposals/Details/Page.js

```diff
@@ -14,38 +14,45 @@ export default ({ viewedProposalDetails, goBackHistory,
   showPurchaseTicketsPage, hasTickets, onVoteOptionSelected, onUpdateVoteChoice,
   newVoteChoice, updateVoteChoiceAttempt, tsDate, text }) =>
 {
-  const { name, token, hasEligibleTickets, voteStatus, voteOptions,
-    voteCounts, creator, timestamp, endTimestamp, currentVoteChoice,
+  const { name, token, voteStatus, proposalStatus, voteOptions, voteCounts,
+    creator, timestamp, endTimestamp, currentVoteChoice, hasEligibleTickets,
     version, quorumMinimumVotes } = viewedProposalDetails;
-  const eligibleTicketCount = viewedProposalDetails.eligibleTickets.length;
+  const eligibleTicketCount = viewedProposalDetails.eligibleTicketCount;
```

Comment:
Given you changed `eligibleTickets` to be _all_ eligible tickets (vs the ones from the wallet) the message is showing a large number of tickets for voting (vs before when it showed the number of tickets _from the wallet_ that are eligible for voting).